Some improvements around tooling and diskspace (#3089)
uyuni-project/uyuni-docs: modules/administration/pages/troubleshooting/tshoot-diskspace.adoc

{productname} monitors free space in specific directories, and has configurable alerts.
For more on space management, see xref:administration:space-management.adoc[].

> Generally, container volumes share space with the host filesystem. When the btrfs filesystem becomes full, it can prevent you from deleting anything.
> To resolve this, you can add a small amount of storage to the filesystem using [command]``btrfs device add`` command. This will allow you to delete the necessary files.
> After that, you can remove the additional storage with [command]``btrfs device delete``. Some more troubleshooting information around this topic can be found here https://www.suse.com/support/kb/doc/?id=000018779
> 

Reviewer: This is just a suggestion.  I'm not sure I completely understood the operations involved and then used wording clear enough.
```suggestion
Generally, container volumes share space with the host filesystem.
When the [systemitem]``btrfs`` filesystem becomes full, it can prevent you from deleting files.
To resolve this, you can add a small amount of storage temporarily to the filesystem using the [command]``btrfs device add`` command.
This will allow you to delete files to gain space for further filesystem maintenance.
When done with the maintenance, you can remove the temporary storage with [command]``btrfs device delete``.
For more information about this topic, see https://www.suse.com/support/kb/doc/?id=000018779.
```

You can recover disk space by removing unused software channels.

> For instructions on how to delete vendor channels, see xref:administration:channel-management.adoc[].
> For instructions on how to delete custom channels, see xref:administration:custom-channels.adoc[].

Reviewer: Drive-by edit:

```suggestion
* For instructions on how to delete vendor channels, see xref:administration:channel-management.adoc[].
* For instructions on how to delete custom channels, see xref:administration:custom-channels.adoc[].
```

{productname} monitors free space in specific directories, and has configurable alerts.
For more on space management, see xref:administration:space-management.adoc[].

Generally, container volumes share space with the host filesystem. When the btrfs filesystem becomes full, it can prevent you from deleting anything.
To resolve this, you can add a small amount of storage to the filesystem using [command]``btrfs device add`` command. This will allow you to delete the necessary files.
After that, you can remove the additional storage with [command]``btrfs device delete``. Some more troubleshooting information around this topic can be found here https://www.suse.com/support/kb/doc/?id=000018779

You can recover disk space by removing unused software channels.

> * For instructions on how to delete vendor channels, see xref:administration:channel-management.adoc[].

Reviewer 2: ```suggestion

* For instructions on how to delete vendor channels, see xref:administration:channel-management.adoc[].
```
